AdaptorsDependencies/src/ivy.xml

```diff
@@ -8,10 +8,44 @@
         <conf name="test" description="Test dependencies" visibility="public" extends="runtime"/>
         <conf name="provided" description="Provided dependencies" visibility="public"/>
     </configurations>
-
+    
     <dependencies defaultconf="runtime">
+        <!-- GAFFER Dependencies -->
+        <dependency org="com.fasterxml.jackson.core" name="jackson-annotations" rev="2.9.6"/> 
```

Comment:
Jackson is made available by the core modules, these would be duplicates.

Reply:
Removed